XLS-0033d-compact-fungible-tokens/README.md

```diff
@@ -67,11 +67,28 @@ The **`CFTokenIssuance`** object represents a single CFT issuance and holds data
 
 ##### 1.2.1.1.1. **`CFTokenIssuance`** Ledger Identifier
 
-The ID of a CFTokenIssuance object, a.k.a. `CFTokenIssuanceID`, is the result of SHA512-Half of the following values, concatenated in order:
+The key of a `CFTokenIssuance` object, is the result of SHA512-Half of the following values, concatenated in order:
 
 * The CFTokenIssuance space key (0x007E).
+* The transaction sequence number.
 * The AccountID of the issuer.
+
+
+The ID of a `CFTokenIssuance` object, a.k.a. `CFTokenIssuanceID`, is a 192-bit integer, concatenated in order:
+
 * The transaction sequence number.
+* The AccountID of the issuer.
+
+```
+┌──────────────────────────┐┌──────────────────────────┐
+│                          ││                          │
+│      Sequence            ││     Issuer AccountID     │
```

Comment:
Let's update the FAQ here to discuss why we're using a `Sequence` number at all -- e.g., so that an MPT can't be recreated after having been deleted (and other reasons)?

Reply:
Sounds good, thanks for adding!